src/components/collection/TreeViewPicker.tsx

```diff
@@ -0,0 +1,111 @@
+import ChevronRightIcon from '@mui/icons-material/ChevronRight';
```

Comment:
Not done but including so the effort isn't lost since we know this is how we want to visualize entities.